mlrun/db/base.py

```diff
@@ -722,7 +722,9 @@ def list_model_endpoints(
         tsdb_metrics: bool = False,
         metric_list: Optional[list[str]] = None,
         top_level: bool = False,
-        mode: Optional[mlrun.common.schemas.EndpointMode] = None,
+        modes: Optional[
+            Union[mm_constants.EndpointMode, list[mm_constants.EndpointMode]]
+        ] = None,
```

Comment:
I would consider keeping only the list option for simplicity, especially since it's an internal API